docs/MVS_Outputs.rst

```diff
@@ -100,21 +100,100 @@ Technical data
+
+        i &\text{: Renewable generation assets 1,2,…}
+
+        gen &\text{: Renewable and non-renewable generation}
+
+        j &\text{: Renewable and non-renewable generation assets 1,2,…}
+
+        w &\text{: Energy carrier weighting factor for output of asset i/j}
+
+
+:Example:
+
+An energy system is composed of a heat and electricity side. Following are the energy flows:
+
+* 100 kWh from a local PV plant
+* 0 kWh local generation for the heat side
+
+This results in:
+
+* A single-vector renewable share of local generation of 0% for the heat sector.
+* A single-vector renewable share of local generation of 100% for the electricity sector.
+* A system-wide renewable share of local generation of 100%.
+
+
+.. _kpi_renewable_factor:
+
+Renewable factor (RF)
 #####################
 
-Describes the share of the MES demand that is supplied from renewable sources.
+Describes the share of the energy influx to the local energy system that is provided from renewable sources.
+This includes both local generation as well as consumption from energy providers.
 
 .. math::
-        RES &=\frac{\sum_i {E_{RES,generation} (i) \cdot w_i}}{\sum_j {E_{generation}(j) \cdot w_j}+\sum_k {E_{grid} (k)}}
+        RF &=\frac{\sum_i {E_{rgen,i} \cdot w_i + RES \cdot E_{grid}}}{\sum_j {E_{gen,j} \cdot w_j}+\sum_k {E_{grid} (k)}}
 
-        \text{with } &i \epsilon \text{[PV,Geothermal,…]}
+        \text{with } rgen &\text{: Renewable generation}
 
-        &j \epsilon \text{[generation assets 1,2,…]}
+        i &\text{: Renewable generation assets 1,2,…}
 
-        &k \epsilon \text{[DSO 1,2…]}
+        gen &\text{: Renewable and non-renewable generation}
 
-** The content of this section was copied from the conference paper handed in to CIRED 2020**
+        j &\text{: Renewable and non-renewable generation assets 1,2,…}
+
+        RES &\text{: Renewable energy share of energy provider}
+
+        k &\text{: Energy provider 1,2…}
+
+        w &\text{: Energy carrier weighting factor for output of asset i/j/k}
+
+:Example:
+
+An energy system is composed of a heat and electricity side. Following are the energy flows:
+
+* 100 kWh from a local PV plant
+* 0 kWh local generation for the heat side
+* 100 kWh consumption from the electricity provider, who has a renewable factor of 50%
+
+Again, the heat sector would have a renewable factor of 0% when considered seperately, and the electricity side would have an renewable factor of 75%. This results in a system-wide renewable share of:
+
+.. math:: RF = \frac{100 kWh(el)\cdot \frac{kWh(eleq)}{kWh(el)}+50 kWh(el)\cdot \frac{kWh(eleq)}{kWh(el)}}
+                {200 kWh(el)\cdot \frac{kWh(eleq)}{kWh(el)}}
+                = 3/4 = \text{75 %}
+
+The renewable factor can, just like the :ref:`kpi_renewable_share_of_local_generation` not indicate how much renewable energy is used in each of the sectors. In the future, it may be possible to dive into this together with the degree of sesector-coupling.
```

Comment:
```suggestion
The renewable factor can, just like the :ref:`kpi_renewable_share_of_local_generation` not indicate how much renewable energy is used in each of the sectors. In the future, it may be possible to dive into this together with the degree of sector-coupling.
```